near-contract-standards/src/non_fungible_token/enumeration.rs

```diff
@@ -0,0 +1,48 @@
+use crate::non_fungible_token::token::Token;
+use near_sdk::json_types::{ValidAccountId, U128};
+
+pub trait NonFungibleTokenEnumeration {
```

Comment:
Just noting that all of these public traits (won't comment on all) have no doc comment. I don't think it matters much, but might be helpful and probably good practice since it's exported

Reply:
Thanks for noting this. I went through and added a couple dozen lines of doc comments. I think we'll still need to comb through this whole crate. I really appreciate you pointing this out. I would like to have docs be an integral part of the code review process.